pkg/rds/adapters.go

```diff
@@ -92,15 +134,15 @@ func (adapter *RDSAdapter) GetSystemInfo() ([]metrics.FlatValue, error) {
 	adapter.Logger().Info("Collecting system info")
 
 	// Refreshes self
-	dbInfo, err := FetchDBInfo(
+	dbInstanceInfo, err := FetchRDSDBInstance(
 		adapter.Config.RDSDatabaseIdentifier,
 		&adapter.AWSClients,
 		context.Background(),
 	)
 	if err != nil {
 		return nil, err
 	}
-	adapter.State.DBInfo = &dbInfo
+	adapter.State.DBInfo.DBInstance = *dbInstanceInfo
```

Comment:
The rest of DBInfo was static information about the EC2 instance type, we don't need to refresh that each tick